recipe/meta.yaml

```diff
@@ -43,3 +42,4 @@ extra:
   recipe-maintainers:
     - mariobuikhuizen
     - jtpio
+    - fcollonval
```

Comment:
cc @fcollonval let me know if it's ok to add you to the list of maintainers

Reply:
Ok for me thanks @jtpio

Reply:
Nice, thanks!